src/main/java/org/assertj/core/internal/Objects.java

```diff
@@ -731,7 +732,13 @@ public boolean areEqualToComparingOnlyGivenFields(Object actual, Object other, S
 	return isEqualToComparingOnlyGivenFields(actual, other, fields).isFieldsNamesEmpty();
   }
 
-  public static class ByFieldsComparison {
+  public boolean assertHasToString(AssertionInfo info, Object actual, Object expected) {
+  assertNotNull(info, actual);
```

Comment:
Hm, sadly as it sounds i am not able to get it right. I am using IntelliJ and have imported assertj-eclipse-formatter.xml as code formatting.